utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -64,7 +62,7 @@ object LicenseFilenamePatterns {
      * given absolute [directory] or in any of its ancestor directories.
      */
     fun getLicenseFileGlobsForDirectory(directory: String): List<String> =
-        getFileGlobsForDirectoryAndAncestors(directory, ALL_LICENSE_FILENAMES)
+        getFileGlobsForParentDirectories(directory, ALL_LICENSE_FILENAMES)
```

Comment:
This KDoc also would need to be updated, to match the new semantic.

Reply:
Thinking about it the renaming was not correct, because the license files within `directory` are still returned and not only the files from ancestor directories.
--> Reverted the renaming and adjusted the KDocs accordingly.